### PR TITLE
YLP-1326 Implement Loop Mode objects upload

### PR DESCRIPTION
**Pull request by CedricPerotto**

### CODE REVIEW

File: data/types/device/alarm/alarm.go [hunk 1]
```diff
@@ -28,7 +28,7 @@ const (
 	Acknowledged            = "acknowledged"
 	Outdated                = "outdated"
 	AlarmCodeMaximumLength  = 64
-	EventIDMaximumLength    = 64
+	GUIDMaximumLength       = 64
```
Comment (Nas84) [inline data/types/device/alarm/alarm.go]: What do you think if we ask Devdroid on the maximun length for this field ?

**Review by Nas84 — COMMENTED** [review]
Looks good , just few questions: 
- should we raise the maximum length on guid (#SeagullV2) ?
- i am not sure did we test the case where the guid is not provided and the eventId is ?

File: data/types/device/alarm/alarm.go (hunk 1 above)
Comment (CedricPerotto) [reply data/types/device/alarm/alarm.go]: We only check length on alarm objects... Either we check it everywhere or we never check it (I would personalty remove this validation).

Comment (Nas84) [reply data/types/device/alarm/alarm.go]: fine to remove this validation, it could cause us some trouble

**Comment by CedricPerotto** [comment]
>     * should we raise the maximum length on guid (#SeagullV2) ?
just dropped it
>     * i am not sure did we test the case where the guid is not provided and the eventId is ?
see tests in data/types/base_test.go (and I just updated some system tests with local run).

**Review by yanngit — APPROVED** [review]

**Pull request merged**